CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+
+#### Directory Layout
+While the exact directory layout for one of our ADX projects can vary on a project-by-project basis, a typical presentation of our projects is as follows:
+```
+.
+├── pre-processing                  # Data provider side source code files and automation code
+│   ├── pre-processing-code         # Source code
+│   │   ├── lambda_function.py      # Code to interact with ADX for creating a dataset and revision
+│   │   └── source_data.py          # Code to acquire dataset and perform necessary data transformation
+│   └── pre-processing-cfn.yaml     # CloudFormation template to setup data provider automation
+├── dataset-description.md          # Dataset description that goes on the ADX listing
+├── product-description.md          # Product description that goes on the ADX listing
+├── init.sh                         # Initialization script to kick-off the automation
+├── README.md
+└── .gitignore
+```
+For more details on the technologies and directory layout used in our ADX products, please visit [Getting started with publishing a data product on AWS Data Exchange](https://github.com/rearc-data/publish-a-data-product-on-aws-data-exchange).
+
+## How can I contribute?
+
+#### Report an Issue/Bug
+If you are experiencing an issue with the ADX product featured in this repository, the best way to contact us would be through [opening a GitHub issue](https://github.com/rearc-data/publish-a-data-product-on-aws-data-exchange/issues) in this repository.
+
+When opening an issue please **be as descriptive as possible**. If relevant please **provide information regarding your use-case, development configuration and environment**. The more specific you can be the easier it will be for us to identify and address the situation.
+
+#### Submit an Improvement/Suggestion
+If you have a suggestion for improving the ADX product included in this repository you are welcomed to [open a GitHub issue](https://github.com/rearc-data/publish-a-data-product-on-aws-data-exchange/issues) in this repository. While we cannot guarantee every suggestion will be implemented into our products, we will get back to you either way.
+
+If you have any general feedback or suggestions for our ADX products, please contact us at data@rearc.io.
+
+#### Pull Request
+One of the major reasons we have made our ADX project repositories public is because we want to better maintain and improve the quality of our data products. We believe our subscribers (you) can play a critical role in this process, and we actively encourage you to fork, branch and open a pull request on this repository. 
```

Comment:
Repeats some of the stuff above.

Reply:
It's no longer repeating prior details, and it's been edited for increased brevity